stories/assets/js/navigationData.js

```diff
@@ -14,7 +14,7 @@ export const getData = (lang) => {
         const submenus = item.submenus
           .map((menu, index) => `<li id="${menu.link.id}-${item.id}" class="${
             index === 0 ? 'active' : ''
-          }"><a href="#" class="sub-link">${menu.link.label}</a></li>`)
+          }"><a href="${menu.external_link?menu.external_link:'#'}" class="${menu.external_link?['sub-link','sub-external'].join(' '):'sub-link'}">${menu.link.label}</a></li>`)
```

Comment:
@Ashish-srij Change this key from `external_link` to `link` or `url`.

Reply:
@shishirphadke We already have URL and link key on navigation data.
The "external link" is as provided document. My suggestion is please don't change this key. Please advice.